### PR TITLE
Added patch for Minecraft.jar to make it run with Java 17 without errors

### PR DESCRIPTION
**Pull request by MrMasrozYTLIVE**
compatibleJavaMajors: Java 8 -> 17

### CODE REVIEW

File: patches/net.minecraft.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                "url": "https://launcher.mojang.com/v1/objects/43db9b498cb67058d2e12d394e6507722e71bb45/client.jar"
+            }
+        },
+        "name": "com.mojang:minecraft:b1.7.3:client"
+    },
+    "name": "Minecraft",
+    "releaseTime": "2011-07-08T00:00:00+02:00",
+    "requires": [
+        {
+            "suggests": "2.9.4-nightly-20150209",
```
Comment (thecatcore) [inline patches/net.minecraft.json]: Babric provides its own lwjgl version, maybe you should set it there as well.

Comment (MrMasrozYTLIVE) [reply patches/net.minecraft.json]: 👍

**Comment by js6pak** [comment]
Babric itself doesn't require java 17, so instead of overriding `net.minecraft.json` couldn't we just add  `17` to `compatibleJavaMajors` in `babric.json`?

**Comment by MrMasrozYTLIVE** [comment]
Default minecraft json has "compatibleJavaMajor" set to java 8 so I think it won't work this way. You can actually just remove compatibleJavaMajor at all and it will launch with any java version

**Pull request closed without merging**